lib/kion/car.go

```diff
@@ -153,7 +155,24 @@ func GetCARByNameAndAccount(host string, token string, carName string, accountNu
 		}
 	}
 
-	return CAR{}, fmt.Errorf("unable to find car %v", carName)
+	return CAR{}, fmt.Errorf("unable to find car %v with account number %v", carName, accountNumber)
+}
+
+// GetCARByNameAndAlias returns a car that matches by name and account alias.
+func GetCARByNameAndAlias(host string, token string, carName string, accountAlias string) (CAR, error) {
+	allCars, err := GetCARS(host, token)
```

Comment:
The `/api/v3/me/cloud-access-role` accepts an `account_alias=?` query parameter as of 3.10.2 (and 3.9.9).  On earlier versions, this parameter is ignored.   You could consider having `GetCARs` pass the alias which could make the API call slightly more efficient when aliases are supported.  And leaving the matching code below would allow things to still work on all versions.

Reply:
Pushed up change to use the query param. Waiting to resolve until tested with updates Kion side.

Reply:
Tests look good, resolving this thread.